04.ls/ls1.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+def list_files
+  current_path = Dir.pwd
+  folder_glob = "#{current_path}/*"
+  files = Dir.glob(folder_glob)
+  files.map { |file| File.basename(file) }
+end
+
+def print_files_in_columns(columns = 3)
+  file_list = list_files
```

Comment:
```suggestion
  files = list_files
```
```suggestion
  columns = Array.new(rows){[]}
```

「〜の配列」という名前を表現したいときは、複数形にすればOKです。（`column_arrays`も同様）